src/stepfunctions/steps/compute.py

```diff
@@ -138,6 +140,7 @@ def __init__(self, state_id, wait_for_completion=True, **kwargs):
             comment (str, optional): Human-readable comment or description. (default: None)
             input_path (str, optional): Path applied to the state’s raw input to select some or all of it; that selection is used by the state. (default: '$')
             parameters (dict, optional): The value of this field becomes the effective input for the state.
+            result_selector (dict, optional): The value of this field becomes the effective result of the state.
```

Comment:
nit: should probably add at least a basic test for every class that exercises the support we are adding here. Or if there are existing tests for optional parameters, update those.

Reply:
Agreed - will update existing tests